Dockerfile

```diff
@@ -10,4 +10,4 @@ WORKDIR $DOCKER_WORKSPACE_PATH/src
 ENV HOME=$DOCKER_WORKSPACE_PATH/.home
 
 # TODO: List project dependencies here
-# RUN pip install ...
\ No newline at end of file
+# RUN pip install ...
```

Comment:
We should add something like
```
COPY requirements.txt .
RUN pip install -r requirements.txt
```

Reply:
Yes, I did that first but then removed it because I did not really see what you did above. Either way, I think that should go almost at the top because otherwise we will have to reinstall all packages every time something above changes

Reply:
Which can be time-consuming on a local machine